CHANGELOG.md

```diff
@@ -7,11 +7,12 @@
 # 1.0.9-rc6 (2020-02-29)
 ## Edge Hub
 ### Bug Fixes
-Hotfix [Linux ARM32 and ARM64 security vulnerability](https://lists.ubuntu.com/archives/ubuntu-security-announce/2020-January/005286.html) by updating base dotnet core runtime images ([fa634ef](https://github.com/Azure/iotedge/commit/fa634efccdc9694e611d97c4dd57a44ad9b742bb))
+* Hotfix [Linux ARM32 and ARM64 security vulnerability](https://lists.ubuntu.com/archives/ubuntu-security-announce/2020-January/005286.html) by updating base dotnet core runtime images ([fa634ef](https://github.com/Azure/iotedge/commit/fa634efccdc9694e611d97c4dd57a44ad9b742bb))
+* Use LogLevel to limit RocksDB LOG file size ([c984f63](https://github.com/Azure/iotedge/commit/c984f6330daec8800e6887f666036df5bdc7ec68))
 
 ## Edge Agent
 ### Bug Fixes
-Hotfix [Linux ARM32 and ARM64 security vulnerability](https://lists.ubuntu.com/archives/ubuntu-security-announce/2020-January/005286.html) by updating base dotnet core runtime images ([fa634ef](https://github.com/Azure/iotedge/commit/fa634efccdc9694e611d97c4dd57a44ad9b742bb))
+* Hotfix [Linux ARM32 and ARM64 security vulnerability](https://lists.ubuntu.com/archives/ubuntu-security-announce/2020-January/005286.html) by updating base dotnet core runtime images ([fa634ef](https://github.com/Azure/iotedge/commit/fa634efccdc9694e611d97c4dd57a44ad9b742bb))
 
```

Comment:
* Use LogLevel to limit RocksDB LOG file size ([c984f63](https://github.com/Azure/iotedge/commit/c984f6330daec8800e6887f666036df5bdc7ec68))